gulpfile.js

```diff
@@ -14,6 +14,57 @@ require('laravel-elixir-vue-2');
  */
 
 elixir(mix => {
-    mix.sass('app.scss')
-       .webpack('app.js');
+    mix
+        .styles(
+            [
+                'slick-carousel/slick/slick.css',
+                'slick-carousel/slick/slick-theme.css',
+                'eonasdan-bootstrap-datetimepicker/build/css/bootstrap-datetimepicker.min.css',
+            ],
+            'public/css/bower.css',
+            './resources/assets/bower_components/'
+        )
+        .styles(
+            [
+                '*.css',
```

Comment:
If there is a third argument to `styles()` then all listed files are assumed relative to that. By default it's `resources/assets/css/`. If they're not in one directory a globstar might work, but I haven't tested it. This could allow for all subdirectories on one directory. Alternatively you could list individual files, like what is done in `scripts()`.